source/features/repo-age.tsx

```diff
@@ -108,14 +108,14 @@ async function init(): Promise<void> {
 	// About a day old or less ?
 	const age = Date.now() - birthday.getTime() < 10e7
 		? fresh[Math.floor(Math.random() * fresh.length)]
-		: `${value} ${unit} old`;
+		: <><strong>{value}</strong> {unit} old</>;
```

Comment:
This 👍 I was going to mention this when I see GitHub's new items but forgot 😅

Reply:
"1 year" what though?

Like the eye icon doesn't stand on its own, neither does "year"